src/Codeception/Test/Loader/Cest.php

```diff
@@ -87,6 +88,7 @@ function ($v) {
                         }
                         $test = new CestFormat($unit, $method, $file);
                         $test->getMetadata()->setCurrent(['example' => $example]);
+                        $test->getMetadata()->setCurrent(['iteration' => ++$iteration]);
```

Comment:
There is no need to introduce a new variable, $k should be fit for this purpose.